EIPS/eip-681.md

```diff
@@ -54,7 +54,7 @@ For the syntax of ENS_NAME, please consult ERC #137 defining Ethereum Name Servi
 
 ### Semantics
 
-`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact.
+`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact. Make sure to respect ERC #831 and add the "pay-" prefix in case you use ENS here.
```

Comment:
@ligi can you please change "ERC #831" to "ERC-831" and make it a link to https://eips.ethereum.org/EIPS/eip-831

Reply:
Oh I see you have followed the style of the draft, however this is the recommendation as per EIP-1.